js-variables-and-numbers_simple-calculator/js/index.js

```diff
@@ -61,24 +86,37 @@ const divideByTwoButton = document.querySelector('[data-js="divide-by-two"]');
 
 increaseByOneButton.addEventListener("click", () => {
   // 7. Increase the value of "operand1" by 1. Log the new value to the console.
+let operantIncrease = operand1 + 1;
```

Comment:
you can also make it shorter with operantIncrease++
it automatically adds one

Reply:
Thank you